content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -35,8 +99,9 @@ Search for FamilyMemberHistories that meet supplied query parameters:
 
 Name      | Required?       | Type          | Description
 ----------|-----------------|---------------|-------------------------------------------------------
-`_id`     | This or patient | [`token`]     | The logical resource id associated with the resource.
-`patient` | This or _id     | [`reference`] | The identity of a subject to list family member history items for. Example: `12345`
+`_id`     | This or patient         | [`token`]     | The logical resource id associated with the resource.
+`patient` | This or _id             | [`reference`] | The identity of a subject to list family member history items for. Example: `12345`
+`status`  | No. Patient is required | [`token`] | The status of the record of the family history of a specific family member.
```

Comment:
I think we just leave this as No

Reply:
https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68R108